src/MainWindow/main_window.h

```diff
@@ -107,6 +107,7 @@ class MainWindow : public QMainWindow, public TopLevelInterface {
 
   void cleanUpDockWidgets(std::vector<QDockWidget*>& dockWidgets);
   void openProject(const QString& project);
+  void openProject(const std::string& projectFile) override;
```

Comment:
if use QString you don't need additional method

Reply:
agree. fixed